File: app/src/main/java/com/forkmang/activity/FaceLogin.java
package com.forkmang.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageView;

import com.forkmang.R;

public class FaceLogin extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_face_login);
        ImageView img_faceid = findViewById(R.id.img_faceid);
        Button BtnReg = findViewById(R.id.BtnReg);


        BtnReg.setOnClickListener(v -> {
            final Intent mainIntent = new Intent(FaceLogin.this, RegisterActivity.class);
            startActivity(mainIntent);
            //finish();
        });

        img_faceid.setOnClickListener(v -> {
            final Intent mainIntent = new Intent(FaceLogin.this, FaceLoginPermission.class);
            startActivity(mainIntent);
            //finish();
        });
    }
}

File: app/src/main/java/com/forkmang/activity/FaceLogin.kt
package com.forkmang.activity

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.forkmang.R

class FaceLogin : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_face_login)

        val imgFaceId: ImageView = findViewById(R.id.img_faceid)
        val btnReg: Button = findViewById(R.id.BtnReg)
        btnReg.setOnClickListener {
            val mainIntent = Intent(this@FaceLogin, RegisterActivity::class.java)
            startActivity(mainIntent)
        }
        imgFaceId.setOnClickListener {
            val mainIntent = Intent(this@FaceLogin, FaceLoginPermission::class.java)
            startActivity(mainIntent)
        }
    }
}